kit/src/elements/file/mod.rs

```diff
@@ -87,9 +87,19 @@ pub fn emit_press(cx: &Scope<Props>) {
     }
 }
 
+pub fn get_file_extension(file_name: String) -> String {
```

Comment:
i see this is only used for `get_text`. if you want this to be a function so you can unit test it, then that's cool. but I think it's a mistake to make `get_text` take a second argument for `file_extension`. if `get_text` needs the file extension, it can call `get_file_extension` on its own. I think this should be changed back before merging.